code/modules/surgery/organs/tails.dm

```diff
@@ -105,3 +105,24 @@
 		H.dna.features["tail_monkey"] = "None"
 		H.dna.species.mutant_bodyparts -= "tail_monkey"
 		H.update_body()
+
+/obj/item/organ/tail/lizard/kobold
+	name = "kobold tail"
+	desc = "A smaller version of a normal lizard tail. Somewhere, no doubt, a lizard hater feels cheated."
+	tail_type = "Kobold"
+	icon_state = "severedkoboldtail"
+
+/obj/item/organ/tail/monkey/Insert(mob/living/carbon/human/H, special = FALSE, drop_if_replaced = TRUE)
```

Comment:
wrong path

Reply:
Kobolds are humans though? Seems like the correct path. Also you should use /human insteaf of /H, no idea why but maintainers demand it

Reply:
he meant that its tail/monkey